src/hdrgen.d

```diff
@@ -2459,7 +2454,15 @@ public:
 
     override void visit(DeclarationExp e)
     {
-        e.declaration.accept(this);
+        if (auto v = e.declaration.isVarDeclaration())
+        {
+            buf.writeByte('(');
+            visitVarDecl(v, false);
+            buf.writeByte(';');
+            buf.writeByte(')');
```

Comment:
Please add comments to the code describing its purpose - because it just looks wrong and does emit incorrect D code.